scripts/teardown-test-registry.sh

```
@@ -0,0 +1,242 @@
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --help|-h)
+            show_usage
+            exit 0
+            ;;
+        --clean-volume)
+            CLEAN_VOLUME="true"
+            shift
+            ;;
+        --no-clean-images)
+            CLEAN_IMAGES="false"
```

Shellcheck (0.10.0)

[warning] CLEAN_IMAGES appears unused. Verify use (or export if used externally).

(SC2034)